crates/api_common/src/utils.rs

```diff
@@ -743,6 +751,14 @@ pub fn sanitize_html_federation_opt(data: &Option<String>) -> Option<String> {
   data.as_ref().map(|d| sanitize_html_federation(d))
 }
 
+pub fn create_login_cookie(jwt: Sensitive<String>) -> Cookie<'static> {
+  let mut cookie = Cookie::new(AUTH_COOKIE_NAME, jwt.into_inner());
+  cookie.set_secure(true);
+  cookie.set_same_site(SameSite::Strict);
+  cookie.set_http_only(true);
```

Comment:
I don't fully understand the interaction with the `jwt` cookie from lemmy-ui and this one.

But in general SameSite=strict has in issue that it is not passed with HTTP GET requests that originate from a different origin.

So for example, if you click this link https://lemmy.ml/post/123456 from github, then samesite=strict cookies will not be passed and the site will represent as logged-out.

Reply:
Didnt know that. Im changing it to set `SameSite::Lax` here, and removing the check for SameSite value in `read_auth_token()`.